lib/BloqadeExpr/src/types.jl

```diff
@@ -139,7 +143,7 @@ nqudits: 1
 """
 Base.@kwdef struct SumOfX <: AbstractTerm
     nsites::Int
-    Ω = 1
+    Ω = 1   # NOTE: Why it has any type?
```

Comment:
we currently let omega to be a function, so it has any type, instead of a type parameter, since this is only called at the interface.